src/js/index.js

```diff
@@ -743,7 +743,11 @@ export const hpe = deepFreeze({
       },
     },
     pad: 'small',
-    margin: 'none',
+    margin: {
+      // bring the overall tabs border behind invidual tab borders
+      vertical: '-2px',
```

Comment:
feels hacky

Reply:
It is a little bit hacky, but the reason we have to do it this way is because the tabs container wraps around the outside of the individual tabs. So, in order the have this border extend inline with the individual tab borders, we have to use a negative margin on the individual tabs. @ericsoderberghp and I chatted about this approach and found it to be better than having to do theming in the extend.

This allows us to achieve this by default (where the tabs bottom border will span the width of its container):
<img width="1039" alt="Screen Shot 2020-06-09 at 1 00 23 PM" src="https://user-images.githubusercontent.com/12522275/84194253-99f0f600-aa51-11ea-827d-95b42b790afb.png">

For use cases (if they arise), where the user does not want this extended border, they can use `alignControls=start` to achieve this, where the controls alignSelf to the start, but the tab content would still be able to span the full width:
<img width="1028" alt="Screen Shot 2020-06-09 at 1 03 50 PM" src="https://user-images.githubusercontent.com/12522275/84194361-c442b380-aa51-11ea-8cb1-29ac48378e4a.png">

Eric noted that it would be fine for the default to be that the border extends the full width.

Reply:
ok

Reply:
Unless you think there would be a better way to move that `Tabs` border to the same level as the other borders. I am open to discussing more grommety ways if you have something in mind!